prepare-toolchain

```diff
@@ -56,40 +56,17 @@ cd swift-source
 git clone https://github.com/apple/swift.git
 swift/utils/update-checkout --clone --tag "swift-${version}-RELEASE"
 
+ln -s ./llvm-project/llvm ./llvm
+ln -s ./llvm-project/clang ./clang
+
 # patch clang
+info "Patching Clang"
 sed -i '' -e 's;__has_include(<CoreServices/CoreServices.h>);__has_include(<FSEvents/FSEvents.h>);g' clang/lib/DirectoryWatcher/DirectoryWatcher.cpp
```

Comment:
path to the DirectoryWatcher has changed to llvm-project/clang/lib/DirectoryWatcher/DirectoryWatcher.cpp